socketio/src/packet.rs

```diff
@@ -118,24 +118,8 @@ impl From<&Packet> for Bytes {
 
         let mut buffer = BytesMut::new();
         buffer.put(string.as_ref());
-        if packet.attachments.as_ref().is_some() {
-            // check if an event type is present
-            let placeholder = if let Some(event_type) = packet.data.as_ref() {
-                format!(
-                    "[{},{{\"_placeholder\":true,\"num\":{}}}]",
-                    event_type,
-                    packet.attachment_count - 1,
-                )
-            } else {
-                format!(
-                    "[{{\"_placeholder\":true,\"num\":{}}}]",
-                    packet.attachment_count - 1,
-                )
-            };
 
-            // build the buffers
-            buffer.put(placeholder.as_ref());
-        } else if let Some(data) = packet.data.as_ref() {
+        if let Some(data) = packet.data.as_ref() {
```

Comment:
these two are not equivalent, what changed?

Reply:
currently only support emit one binary payload, if we want to emit multiple binary payload and keep the same order user emit, placeholders are hard to replace in `packet.rs`. so this logic is moved to `socket.rs`